frontend/src/locales/de.json

```diff
@@ -8,6 +8,10 @@
             "removeColumn": "Leere Spalte entfernen"
           }
         },
+        "storyboard": {
+          "controls": "Aktionen",
+          "reorder": "Umsortieren"
```

Comment:
This is only for screen readers, right?
Otherwise I'd propose to call this "Reihenfolge ändern". But if it is supposed to describe only a single currently focused drag handle, this should be fine (or maybe "Verschieben" could be more standard German).

Reply:
```suggestion
          "reorder": "Verschieben"
```